### PR TITLE
add attrs/aria-label to navigation element within KBreadcrumbs

### PR DESCRIPTION
**Pull request by marcellamaki**
## Description
Per conversation on Slack, I've added attributes binding and an aria-label option to `KBreadcrumbs` so that it is easier for us to use a label on the <nav> element, which is where it should be.

## Steps to test

1. See [corresponding kolibri PR](https://github.com/learningequality/kolibri/pull/9663) for testing. Successful implementation should have the aria label display on the breadcrumb elements

### Does this introduce any tech-debt items?
Is this clear enough? would props be better?

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change has been added to the `changelog`

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## Post-merge updates and tracking

- [ ] Learning Platform update PR is submitted
- [ ] Learning Platform update PR is merged
- [ ] Studio update PR is submitted
- [ ] Studio update PR is merged
- [ ] Data Portal update PR is submitted
- [ ] Data Portal update PR is merged

### CODE REVIEW

File: lib/KBreadcrumbs.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
     v-show="showSingleItem || crumbs.length > 1"
     :class="{ 'breadcrumbs-collapsed': collapsedCrumbs.length }"
   >
-    <nav class="breadcrumbs">
+    <nav class="breadcrumbs" v-bind="$attrs" :aria-label="$attrs.ariaLabel">
```
Comment (rtibbles) [inline lib/KBreadcrumbs.vue:7]: I think we should also add `inheritAttrs: false` to the component as well? https://v2.vuejs.org/v2/api/#inheritAttrs

Comment (marcellamaki) [reply lib/KBreadcrumbs.vue:7]: Like we have in the `TextTruncatorCss` component? https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/TextTruncatorCss.vue#L42

Comment (rtibbles) [reply lib/KBreadcrumbs.vue:7]: Yes, it's possible it's not actually required, but this avoids the `$attrs` being erroneously applied to the containing div.

Comment (marcellamaki) [reply lib/KBreadcrumbs.vue:7]: @rtibbles added

**Review by rtibbles — APPROVED** [review]

**Comment by MisRob** [comment]
@marcellamaki Gentle reminder that it'd be good to mention this in the changelog

**Comment by MisRob** [comment]
@marcellamaki Regarding the changelog, I was just opening a PR to `develop`, so I thought I could sneak the update in: https://github.com/learningequality/kolibri-design-system/pull/363

**Comment by marcellamaki** [comment]
@MisRob thank you!

**Pull request merged**